Modules/System Tests/Translation/src/TranslationTestTableTwo.Table.al

```diff
@@ -0,0 +1,41 @@
+﻿// // ------------------------------------------------------------------------------------------------
+// // Copyright (c) Microsoft Corporation. All rights reserved.
+// // Licensed under the MIT License. See License.txt in the project root for license information.
+// // ------------------------------------------------------------------------------------------------
+
+//TODO: Enable in BC19 and add to License
+// table 137122 "Translation Test Table Two"
```

Comment:
looks like something went wrong in the merge and you pulled in some unrelated things